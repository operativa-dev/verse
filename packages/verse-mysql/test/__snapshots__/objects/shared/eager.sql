-- Executing SQL: Parameters: []
select `t2`.`AlbumId`, `t2`.`Title`, `t2`.`ArtistId`, `t4`.`Id`, `t4`.`Name`, `t6`.`AlbumId`, `t6`.`Title`, `t6`.`ArtistId`, `t8`.`Id`, `t8`.`Name`
from `Album` as `t2` 
inner join `Artist` as `t4` on `t2`.`ArtistId` = `t4`.`Id` 
left join `Album` as `t6` on `t4`.`Id` = `t6`.`ArtistId` 
inner join `Artist` as `t8` on `t6`.`ArtistId` = `t8`.`Id`
order by `t4`.`Id`

