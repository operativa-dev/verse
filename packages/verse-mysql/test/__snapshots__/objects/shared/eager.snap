[
  {
    "albumId": 12,
    "artist": {
      "albums": [
        {
          "albumId": 10,
          "artist": [Circular],
          "artistId": 1,
          "title": "The Sixteen Men of Tain",
        },
        {
          "albumId": 11,
          "artist": [Circular],
          "artistId": 1,
          "title": "Atavachron",
        },
        [Circular],
      ],
      "id": 1,
      "name": "Allan Holdsworth",
    },
    "artistId": 1,
    "title": "Metal Fatigue",
  },
  {
    "albumId": 11,
    "artist": {
      "albums": [
        {
          "albumId": 10,
          "artist": [Circular],
          "artistId": 1,
          "title": "The Sixteen Men of Tain",
        },
        [Circular],
        {
          "albumId": 12,
          "artist": [Circular],
          "artistId": 1,
          "title": "Metal Fatigue",
        },
      ],
      "id": 1,
      "name": "Allan Holdsworth",
    },
    "artistId": 1,
    "title": "Atavachron",
  },
  {
    "albumId": 10,
    "artist": {
      "albums": [
        [Circular],
        {
          "albumId": 11,
          "artist": [Circular],
          "artistId": 1,
          "title": "Atavachron",
        },
        {
          "albumId": 12,
          "artist": [Circular],
          "artistId": 1,
          "title": "Metal Fatigue",
        },
      ],
      "id": 1,
      "name": "Allan Holdsworth",
    },
    "artistId": 1,
    "title": "The Sixteen Men of Tain",
  },
  {
    "albumId": 12,
    "artist": {
      "albums": [
        {
          "albumId": 10,
          "artist": [Circular],
          "artistId": 1,
          "title": "The Sixteen Men of Tain",
        },
        {
          "albumId": 11,
          "artist": [Circular],
          "artistId": 1,
          "title": "Atavachron",
        },
        [Circular],
      ],
      "id": 1,
      "name": "Allan Holdsworth",
    },
    "artistId": 1,
    "title": "Metal Fatigue",
  },
  {
    "albumId": 11,
    "artist": {
      "albums": [
        {
          "albumId": 10,
          "artist": [Circular],
          "artistId": 1,
          "title": "The Sixteen Men of Tain",
        },
        [Circular],
        {
          "albumId": 12,
          "artist": [Circular],
          "artistId": 1,
          "title": "Metal Fatigue",
        },
      ],
      "id": 1,
      "name": "Allan Holdsworth",
    },
    "artistId": 1,
    "title": "Atavachron",
  },
  {
    "albumId": 10,
    "artist": {
      "albums": [
        [Circular],
        {
          "albumId": 11,
          "artist": [Circular],
          "artistId": 1,
          "title": "Atavachron",
        },
        {
          "albumId": 12,
          "artist": [Circular],
          "artistId": 1,
          "title": "Metal Fatigue",
        },
      ],
      "id": 1,
      "name": "Allan Holdsworth",
    },
    "artistId": 1,
    "title": "The Sixteen Men of Tain",
  },
  {
    "albumId": 12,
    "artist": {
      "albums": [
        {
          "albumId": 10,
          "artist": [Circular],
          "artistId": 1,
          "title": "The Sixteen Men of Tain",
        },
        {
          "albumId": 11,
          "artist": [Circular],
          "artistId": 1,
          "title": "Atavachron",
        },
        [Circular],
      ],
      "id": 1,
      "name": "Allan Holdsworth",
    },
    "artistId": 1,
    "title": "Metal Fatigue",
  },
  {
    "albumId": 11,
    "artist": {
      "albums": [
        {
          "albumId": 10,
          "artist": [Circular],
          "artistId": 1,
          "title": "The Sixteen Men of Tain",
        },
        [Circular],
        {
          "albumId": 12,
          "artist": [Circular],
          "artistId": 1,
          "title": "Metal Fatigue",
        },
      ],
      "id": 1,
      "name": "Allan Holdsworth",
    },
    "artistId": 1,
    "title": "Atavachron",
  },
  {
    "albumId": 10,
    "artist": {
      "albums": [
        [Circular],
        {
          "albumId": 11,
          "artist": [Circular],
          "artistId": 1,
          "title": "Atavachron",
        },
        {
          "albumId": 12,
          "artist": [Circular],
          "artistId": 1,
          "title": "Metal Fatigue",
        },
      ],
      "id": 1,
      "name": "Allan Holdsworth",
    },
    "artistId": 1,
    "title": "The Sixteen Men of Tain",
  },
]